docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,27 @@
+# How to add self-signed certificate to trusted
+
+To manually add a self-signed certificate to be trusted by Linux system:
+
+Create a `/usr/local/share/ca-certificates/` directory if it does not exist on your computer:
```

Comment:
Have you actually tested these instructions? On Pi and Ubuntu?

Reply:
I actually have :)